tests/test_api.py

```diff
@@ -526,16 +529,15 @@ def test_get_dates_for_course_query_counts(self, has_schedule, pass_user_object,
                 course_id=self.course.id, user=user, schedule=schedule
             )
 
-        # Second time, the request cache eliminates all querying (sometimes)...
-        # If a schedule is not provided, we will get the schedule to avoid caching outdated dates
```

Comment:
... I'm not sure if it's dangerous that I'm altering this particular behavior, i.e. if there's something that assumes schedules are never cached.